lib/facility_builder.rb

```diff
@@ -0,0 +1,92 @@
+class FacilityBuilder
+
```

Comment:
No space on line 2, but DO want a space between the attr_readers and the def initialize